Modules/Sources/JetpackStats/Utilities/Modifiers/ScrollOffsetModifier.swift

```diff
@@ -0,0 +1,28 @@
+import SwiftUI
+
+@available(iOS 18.0, *)
+struct ScrollOffsetModifier: ViewModifier {
+    @Binding var isScrolled: Bool
+
+    func body(content: Content) -> some View {
+        content
+            .onScrollGeometryChange(for: Bool.self) { geometry in
+                return (geometry.contentOffset.y - 20) > -geometry.contentInsets.top
+            } action: { _, newValue in
+                if isScrolled != newValue {
+                    isScrolled = newValue
+                }
+            }
+    }
+}
+
+extension View {
+    @ViewBuilder
+    func trackScrollOffset(isScrolling: Binding<Bool>) -> some View {
```

Comment:
If I'm not mistaken, the `ScrollOffsetModifier` checks whether the scroll view _has been scrolled_, not whether it's currently scrolling? If that's the case, the name `isScrolling` might be a bit misleading?

Reply:
Yeah, I forgot to update it. Done.